### DISCUSSION TITLE
allow --reject option while updating dependencies

### DISCUSSION THREAD

**Discussion by vsramalwan** [pnpm/pnpm#3222] [Ideas]
`pnpm update` could support an equivalent option of `ncu --reject` to not upgrade a small set of dependencies. This is quite helpful for teams updating their dependencies frequently.

**Comment by zkochan** [comment]
I think this is useful in several cases. Sometimes you want to prohibit updates to specific version. For instance, a package may change its license in a specific version, so you might want to disallow updates to that version.

Prior art. Renovate has package rules to declare allow ranges for updates:

https://github.com/pnpm/pnpm/blob/d7a66690378ce762940ea72daec9fa1141608fe3/renovate.json#L75-L104
